qulice-checkstyle/src/test/resources/com/qulice/checkstyle/LocalVariableNames.java

```diff
@@ -7,6 +7,7 @@
  * Simple.
  * @version $Id$
  * @author John Smith (john@example.com)
+ * @checkstyle FinalLocalVariable (51 lines)
```

Comment:
@jrdalpra apologies, this is suppression of `FinalLocalVariable` not `FinalLocalVariableName`! I've confused these two. 
So instead of this suppression you can just modify this variable, so that it couldn't be final.